tracker/mtracker.py

```diff
@@ -25,17 +25,48 @@ def get_all_user_requests():
         })
 
 @app.route('/api/v1/users/requests/<requestId>', methods = ['GET'])
-def get_a_request_for_user():
-    return ''
+def get_a_request_for_user(requestId):
+    for single_request in requests:
+        if single_request.get(id) == requestId:
+            return jsonify({'request': single_request})
+
+    return jsonify({
+        'status': 'Fail',
+        'message': 'That request doesnot exist'
+    })
 
 @app.route('/api/v1/users/requests', methods =['POST'])
 def create_a_request():
-    
-    return ''
+    request_data = request.get_json()
+    employeeName = request_data.get('employeeName')
+    requestId = len (requests) +1
+    description = request_data.get('description')
+    category = request_data.get('category')
+    requestDate = request_data.get('requestDate')
+
+
+    if not employeeName or employeeName == "" or employeeName == type (int):
+        return jsonify({'message':'Employee Name is required'})
+
+    if not description or description ==" ":
+        return jsonify({'message': 'Resquest description is required'})
+        
+    if not category or category=='':
+        return jsonify({'message':'Request Category is required'})
+
+    if not requestDate or requestDate =='':
+        return jsonify({'message':'When was the request logged?'})
+
+    new_request =Request(requestId, description,employeeName, category, requestDate)
+    requests.append(new_request)
+
+    return jsonify({'message':'Hello {employeeName}! Request succesfully created'})
+
 
 @app.route('/api/v1/users/requests/<requestId>', methods =['PUT'])
 def modify_a_request():
-    return ''
```

Comment:
make this method, do nothing otherwise it will return an indetation error for the if statement below.
i.e 
```python
@app.route('/api/v1/users/requests/<requestId>', methods =['PUT'])
 def modify_a_request():
     pass
```

Reply:
thanks @dennisja , I made the recommended changes and also activated my virtualenv